srml/session/src/lib.rs

```diff
@@ -356,9 +358,40 @@ mod tests {
 	pub struct Test;
 	impl consensus::Trait for Test {
 		type Log = DigestItem;
-		type SessionKey = UintAuthorityId;
+		type SessionKey = TestSessionKey;
 		type InherentOfflineReport = ();
 	}
+
+	#[derive(Clone, Debug, Default, PartialEq, Eq, Encode, Decode, Serialize, Deserialize)]
+	pub struct TestSessionKey(u64, u64);
+
+	impl Into<sr25519::Public> for TestSessionKey {
+		fn into(self) -> sr25519::Public {
+				let bytes: [u8; 32] = U256::from(self.0).into();
```

Comment:
Why go through `U256` and not `H256`? Won't you end up with possible endianess issues?

Reply:
I copied that from `testing.rs`. Changed it there too.